.github/workflows/python-package.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
@isilber Python 3.8 has reached end-of-life status back in Oct 2024 - suggest removing.

Also, I'd suggest to test just on a lowest and highest versions (3.9 and 3.12) - as it seems very unlikely that if something works on both, it will choke on a version in between.

Reply:
Makes sense, @slayoo, thank you! Revised accordingly.